tests/e2e-pw/global-setup.ts

```diff
@@ -75,6 +75,52 @@ const loginAsCustomer = async ( config: FullConfig ) => {
 	await browser.close();
 };
 
+const prepareAttributes = async ( config: FullConfig ) => {
+	const { baseURL, userAgent } = config.projects[ 0 ].use;
+
+	// Specify user agent when running against an external test site to avoid getting HTTP 406 NOT ACCEPTABLE errors.
+	const contextOptions = { baseURL, userAgent };
+
+	// Create browser, browserContext, and page for customer and admin users
+	const browser = await chromium.launch();
+	const context = await browser.newContext( contextOptions );
+	const page = await context.newPage();
+
+	await page.goto( `/wp-admin` );
+	await page.fill( 'input[name="log"]', admin.username );
+	await page.fill( 'input[name="pwd"]', admin.password );
+	await page.click( 'text=Log In' );
+
+	/*
+	 * Intercept the dialog event.
+	 * This is needed because when the regenerate
+	 * button is clicked, a dialog is shown.
+	 */
+	page.on( 'dialog', async ( dialog ) => {
+		await dialog.accept();
+	} );
+
+	await page.goto( '/wp-admin/admin.php?page=wc-status&tab=tools' );
+
+	await page.click( '.regenerate_product_attributes_lookup_table input' );
+
+	await context.close();
+	await browser.close();
+
+	/*
+	 * Note that the two commands below are intentionally
+	 * duplicated as we need to run the cron task twice as
+	 * we need to process more than 1 batch of items.
+	 */
+	await cli(
+		`npm run wp-env run tests-cli wp action-scheduler run --hooks="woocommerce_run_product_attribute_lookup_regeneration_callback"`
+	);
+
+	await cli(
+		`npm run wp-env run tests-cli wp action-scheduler run --hooks="woocommerce_run_product_attribute_lookup_regeneration_callback"`
+	);
```

Comment:
While doing the review, this section caught my attention every time I read over it. It's OK to keep it for now, but I wonder if there isn't an alternative way to accomplish this.